neuro_san/http_sidecar/logging/http_logger.py

```diff
@@ -92,5 +99,6 @@ def prepare_filter(self, metadata: Dict[str, Any]):
         """
         Prepare logging filter using request metadata,
         """
-        metadata["source"] = HttpLogger.HTTP_LOGGER_NAME
-        LogContextFilter.log_context.set(metadata)
+        use_metadata: Dict[str, Any] = copy.copy(self.base_metadata)
+        use_metadata.update(metadata)
+        LogContextFilter.log_context.set(use_metadata)
```

Comment:
Update dictionary to be used by logger filter with actual meta-data coming from request being processed.
Now this will work even if incoming meta-data is empty.